cleanlab/rank.py

```diff
@@ -553,3 +553,30 @@ def get_confidence_weighted_entropy_for_each_label(
     label_quality_scores = np.log(label_quality_scores + 1) / clipped_scores
 
     return label_quality_scores
+
+
+def find_top_issues(quality_scores: np.ndarray, top: int = 20) -> np.ndarray:
```

Comment:
```suggestion
def find_top_issues(quality_scores: np.ndarray, *, top: int = 20) -> np.ndarray:
```

Reply:
this suggestion would make it easier to update signature in the future without breaking API